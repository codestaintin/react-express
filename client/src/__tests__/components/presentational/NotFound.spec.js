import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import NotFound from '../../../components/Common/NotFound';


describe('<NoutFound>', () => {
  let shallowComponent;
  beforeEach(() => {
    shallowComponent = shallow(<NotFound />);
  });

  it('renders the NotFound component', () => {
    const tree = toJson(shallowComponent);
    expect(tree).toMatchSnapshot();
  });
});