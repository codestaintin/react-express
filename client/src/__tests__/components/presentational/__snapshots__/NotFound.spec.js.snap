// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoutFound> renders the NotFound component 1`] = `
<div
  className="not-found d-flex flex-row align-items-center"
>
  <div
    className="container"
  >
    <div
      className="row justify-content-center"
    >
      <div
        className="col-md-12 text-center"
      >
        <span
          className="display-1 d-block"
        >
          404
        </span>
        <div
          className="mb-4"
        >
          The page you are looking for was not found.
        </div>
      </div>
    </div>
  </div>
</div>
`;
